caffe2/serialize/inline_container.cc

```diff
@@ -624,6 +642,15 @@ PyTorchStreamWriter::PyTorchStreamWriter(
   setup(archive_name_);
 }
 
+PyTorchStreamWriter::PyTorchStreamWriter(
```

Comment:
Can't this have a default value for the seek_fn so that we can remove the overload above?